deploy/csi-operator.yaml

```diff
@@ -981,7 +1000,12 @@ spec:
           hostPath:
             path: /var/lib/kubelet/
             type: Directory
-        - name: iscsiadm-bin
+        - name: chroot-iscsiadm
+          configMap:
+            defaultMode: 0755
+            name: openebs-cstor-csi-iscsiadm
+        - name: host-root
           hostPath:
-            path: /sbin/iscsiadm
-            type: File
+            path: /
```

Comment:
mounting the root not in readonly mode seems to be little risky and it is the responsibility of the application running inside the conaitner, that it is not messing up with the data and corrupting anything.